sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1352,7 +1352,7 @@ static String getExtractDestinationUri(String extractDestinationDir) {
      * Choose the frequency at which file writes are triggered.
      *
      * <p>This is only applicable when the write method is set to {@link Method#FILE_LOADS}, and
-     * only when writing a bounded {@link PCollection}.
+     * only when writing an unbounded {@link PCollection}.
```

Comment:
It seems like the original comment is correct since this only applies to non streaming insert mode as can be seen here:
https://github.com/javier/beam/blob/6bd20f4f70bd8e51ad494cde51660669be152067/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java#L1588

Reply:
I might be understanding something wrong here, but I am getting this error:

> [ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.4.0:java (default-cli) on project data-specialist: An exception occured while executing the Java class. null: InvocationTargetException: Triggering frequency or number of file shards can be specified only when writing an unbounded PCollection via FILE_LOADS, but: the collection was BOUNDED and the method was FILE_LOADS -> [Help 1]

It says I cannot write a bounded collection and use triggering, that the collection must be unbounded, which I think is the opposite to what the documentation states.

My code is 

```
.withSchema(ActivityRecord.asBQSchema())
                    .withMethod(BigQueryIO.Write.Method.FILE_LOADS)
                    .withTriggeringFrequency(Duration.standardMinutes(1))
                    .withCreateDisposition(BigQueryIO.Write.CreateDisposition.CREATE_IF_NEEDED)
                    .withWriteDisposition(BigQueryIO.Write.WriteDisposition.WRITE_APPEND)

```
Thanks!